commons/pac-batch-commons/src/main/java/com/tmobile/pacman/commons/utils/NotificationPermissionUtils.java

```diff
@@ -14,19 +15,18 @@
 import java.util.Map;
 
 public class NotificationPermissionUtils {
-    private static final String NOTIFICATION_URL ="notification.lambda.function.url" ;
+    private static final String NOTIFICATION_URL = "notification.lambda.function.url";
     static String SUBJECT = "Insufficient permissions";
 
     static String opsEventName = "Permission denied for %s";
     private static PaladinAccessToken accessToken;
     private static final Logger LOGGER = LoggerFactory.getLogger(NotificationPermissionUtils.class);
-    public static void triggerNotificationsForPermissionDenied(List<PermissionVH> permissionVHList,String cloudType) {
+
+    public static void triggerNotificationsForPermissionDenied(List<PermissionVH> permissionVHList, String cloudType) {
```

Comment:
The function name needs to be singular notification since we are sending exactly 1 notification

Reply:
Addressed in commit